package.json

```diff
@@ -21,33 +21,40 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
```

Comment:
I favour pre-commit over pre-push hooks for this kind of things, but  yes agree on this. 💯